src/utils/storage.js

```diff
@@ -0,0 +1,12 @@
+
+export function getLocalStorage (key) {
```

Comment:
getFromLocalStorage

Reply:
setToLocalStorage?

Reply:
```suggestion
export function setLocalStorage (key) {
```
setToLocalStorage ?

Reply:
setToLocalStorage ?

Reply:
export const NAVIGATOR_PERMISSION_DENIED_ERROR = 1
export const NAVIGATOR_POSITION_UNAVAILABLE_ERROR = 2
export const NAVIGATOR_TIMEOUT_ERROR = 3

Is this correct now?